Add mention to the `<Version />` component in `WRITING.md` (#1536)
withastro/docs: WRITING.md

<Since v="0.24.0" />
```

### Version Component

> Sometimes it may be useful to display the latest version of a package in a page. For this, you can use the `<Version />` component, which takes the `pkgName` property that must receive a valid package name in the npm registry as value.

Reviewer: A bit of a language nit, but I hope this is a tiny bit clearer.

```suggestion
Sometimes it may be useful to display the latest version of a package on a page. For this, you can use the `<Version />` component, which must receive a valid package name from the npm registry as its `pkgName` prop.
```

```md
Astro's latest version:  <Version pkgName="astro" />
```

This will render **Astro's latest version: v1.2.1**.

> The `<Version />` component is currently used in our Integrations pages as a way to have the Integrations pages versions always up-to-date without having to merge changes to these pages directly. It's worth noting that this component is only updated at build-time, thus package's versions will not change till the site is rebuilt, be it manually or because a new PR was merged into main.

Reviewer: Some more small language nits here:

```suggestion
The `<Version />` component is currently used in our Integrations pages as a way to keep each integration’s version up-to-date without having to merge changes to these pages directly. It's worth noting that this component is only updated at build-time, thus a package's version will not change until the site is rebuilt, be it manually or because a new PR was merged into main.
```
